STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/inc/dm_1_model.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+/********************************************************************
+ * This code was auto-generated from dm v1 DL and shall not be
```

Comment:
Would the idea be that this code is generated every time a new module is downloaded to the device?

Reply:
No, It shall be autogenerated when a developer decide to implement this interface. The package that the device will download contains this file compiled in the binary.